morph.go

```diff
@@ -72,6 +72,7 @@ func SetSatementTimeoutInSeconds(n int) EngineOption {
 }
 
 // NewFromConnURL creates a new instance of the migrations engine from a connection url
+// This function creates a *sql.DB instance and it won't be closed by the Close() method.
```

Comment:
Well, that should not be done. :) I'd suggest that we remove this API altogether and only keep the `NewWithDriverAndSource` (or even rename it to just `New` to keep things simple) so that the user has to always pass an explicit driver instance. This way, they have access to the DB instance passed and can close it at their convenience.

Reply:
Yep, let's do that. I am inclined to keep `drivers.Connect` to use the utility for CLI commands. Though this is not scoped to this comment but slightly related.